tests/TypeParseTest.php

```diff
@@ -860,6 +861,15 @@ public function testEnumWithoutSpaces(): void
         $this->assertSame($resolved_type->getId(), $docblock_type->getId());
     }
 
+    public function testLongUtf8LiteralString(): void
+    {
+        $string = "АаБбВвГгДдЕеЁёЖжЗзИиЙйКкЛлМмНнОоПпРрСсТтУуФфХхЦцЧчШшЩщЪъЫыЬьЭэЮюЯя";
```

Comment:
I would suggest to double the string, so that it's longer than 80 (multibyte) characters. It'll make sure the string is actually truncated.

Reply:
Well that's exactly what I do below)

Reply:
Oops, missed that.